pkg/server/commands/write_test.go

```diff
@@ -41,7 +42,7 @@ func TestValidateNoDuplicatesAndCorrectSize(t *testing.T) {
 		}
 	}
 
-	cmd := NewWriteCommand(mockDatastore, logger)
+	cmd := NewWriteCommand(mockDatastore, logger, nil)
```

Comment:
if for some reason `cmd.validateNoDuplicatesAndCorrectSize` starts using `typesystem` this whole set of tests will panic. I think it makes sense to change the signature of `validateNoDuplicatesAndCorrectSize` to

```golang
func validateNoDuplicatesAndCorrectSize(deletes []*openfgapb.TupleKey, writes []*openfgapb.TupleKey, maxTuplesPerwrite int) error {
```

Reply:
That's a good point. That method doesn't need to be an instance method (instance of `WriteCommand`).

Reply:
Refactored 👍